src/main/java/dinkplugin/notifiers/data/GambleNotificationData.java

```diff
@@ -0,0 +1,13 @@
+package dinkplugin.notifiers.data;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+import java.util.List;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class GambleNotificationData extends NotificationData {
+    int gambleCount;
+    List<SerializedItemStack> loot;
```

Comment:
```suggestion
    List<SerializedItemStack> items;
```

for consistency with clue/loot notifiers